src/main/java/org/dsmhack/config/AppUserDetailsService.java

```diff
@@ -22,10 +22,10 @@
     @Override
     public UserDetails loadUserByUsername(String userGuid) throws UsernameNotFoundException {
         LoginToken securityCode = loginTokenRepository.findByUserGuid(userGuid);
-        boolean isTokenExpired = securityCode.getTokenExpDate().before(Timestamp.valueOf(LocalDateTime.now()));
+        boolean credentialsNonExpired = LocalDateTime.now().isBefore(securityCode.getTokenExpDate());
```

Comment:
LocalDateTime includes timezone info (I think). Would it be less error prone to work in utc or gmt?

Reply:
At some point we should probably pick a time done that matters. On that note, I wonder what it defaults to.